path_sequence_planner/include/path_sequence_planner/path_sequence_planner.h

```diff
@@ -27,20 +27,23 @@ namespace path_sequence_planner
 class PathSequencePlanner
 {
 public:
+  PathSequencePlanner() = default;
+  virtual ~PathSequencePlanner() = default;
```

Comment:
In a couple places, you have added or removed constructors and destructors.  Is there any particular reason for this?  In particular, explicitly declaring the default constructor in a non-inherited class seems odd.  (Perhaps my C++ naivety is shining through here.)

Reply:
The compiler might (probably?) create these by default, but just an old habit of always defining constructors. I believe it's important for base classes to have virtual destructors defined as well

Reply:
I don't have a personal vendetta against unnecessarily defined constructors or destructors, and I don't have an organization coding standard to point to, either.

However, I can confirm a) the compiler will always define the constructor and destructor for a class if the author does not, b) a subclass constructor will always call a base class constructor before the function body, even if not specified, and c) after exiting, a subclass destructor will always call the base class destructor, even if not specified.